sw/device/tests/aon_timer_wdog_bite_reset_test.c

```diff
@@ -33,8 +32,8 @@ static void config_wdog(const dif_aon_timer_t *aon_timer,
   uint32_t bite_cycles =
       aon_timer_testutils_get_aon_cycles_from_us(bite_time_us);
 
-  LOG_INFO("Wdog will bark after %u us and bite after %u us", bark_time_us,
-           bite_time_us);
+  LOG_INFO("Wdog will bark after %u us and bite after %u us",
```

Comment:
Worth considering this approach for printing u64 (HMAC testutils):
```
  // 64bit formatting is not supported, so split into hi and lo hex 32bit                                                                                                     
  // values. These should appear as 64bit hex values in the debug output.                                                                                                     
  CHECK(expected_sent_bits == sent_bits,
        "Message length mismatch. "
        "Expected 0x%08x%08x bits but got 0x%08x%08x bits.",
        (uint32_t)(expected_sent_bits >> 32), (uint32_t)expected_sent_bits,
        (uint32_t)(sent_bits >> 32), (uint32_t)sent_bits);
```

Reply:
Good Idea, I do it in a follow PR. Thanks!